libgcc/config.host

```diff
@@ -475,8 +478,8 @@ aarch64-*-mingw*)
 	else
 	  tmake_dlldir_file="i386/t-dlldir-x"
 	fi
-	tmake_file="${tmake_file} ${tmake_eh_file} ${tmake_dlldir_file}" # i386/t-chkstk
-        tmake_file="${tmake_file} i386/t-slibgcc-cygming i386/t-cygming i386/t-mingw32 t-dfprules"
+	tmake_file="${tmake_file} ${tmake_eh_file} ${tmake_thr_file} ${tmake_dlldir_file}"
+	tmake_file="${tmake_file} i386/t-slibgcc-cygming i386/t-slibgcc-mingw i386/t-cygming i386/t-mingw32 t-dfprules t-crtfm" # i386/t-chkstk
```

Comment:
I am not sure if fast math (`t-crtfm`) is supported on aarch64. @ZacWalk please let me know how to test it, or I can remove this chage from this PR.

Reply:
undefined references for __gthr-win32* have been fixed in https://github.com/ZacWalk/gcc-woarm64/commit/c203c40a5358c06523a947de5360bbbfeed1dc93

when is t-crtfm needed? why has t-chkstk been moved to another line?

Reply:
I second that question about `t-crtfm`. Note that this PR is rather RFC than a real intention to merge those changes.

The `t-chkstk` was moved because it semantically bellongs to that group of headers, see how x86\_64 is handled.

Reply:
Maybe it is better to change it to draft

Reply:
i386/t-chkstk contains stack probing implementation only for x64 and not needed anymore for aarch64 mingw